public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,40 @@
+# list
+
+List file structure of a repo including <abbr>data artifacts</abbr>
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outputs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Url to the repo. Can be either local fs path or git+ssh/https url
+  target           Path to directory within the repository to list outputs for.
+
+optional arguments:
+  --outputs-only   Show only dvc outputs.
+  --rev [REV]      Git revision (e.g. branch, tag, SHA)
+```
+
+## Description
+
+The command list files and <abbr>data artifacts</abbr> into pointed repo.
+
+## Options
+
+- `--outputs-only` - Show only <abbr>data artifacts</abbr>.
+
+- `-R`, `--recursive` - Recursively traverse the repo.
+
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash) of the repository to download
+  the file or directory from. The tip of the default branch is used by default
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information. when this option is
```

Comment:
`Works only for remote repos` ?

Reply:
If the repo is on the filesystem pointing `rev` throwing an error. `dvc list` lists files through fs walk rather then `ls-tree` so for the current implementation needs to checkout the repo
https://github.com/iterative/dvc/pull/3246#discussion_r373534682

Reply:
but this `-v` option, verbose ... It should be working for any combination of params